draft-ietf-oauth-status-list.md

```diff
@@ -223,16 +227,21 @@ TBD
 
 # Referenced Token {#referenced-token}
 
+## Status Claim {#status-claim}
+
+By including a "status" claim in a JWT, the issuer of the JWT declares that the credential is referencing a mechanism to retrieve status information about this credential. The claim contains members used to reference to a status list as defined in this specification. Other members of the "status" object may be defined because status list means to check the status of a credential. This is analogous to "cnf" claim in Section 3.1 of {{RFC7800}} in which different authenticity confirmation methods can be included.
```

Comment:
```suggestion
By including a "status" claim in a Referenced Token, the issuer is referencing a mechanism to retrieve status information about this Referenced Token. The claim contains members used to reference to a status list as defined in this specification. Other members of the "status" object may be defined by other specifications. This is analogous to "cnf" claim in Section 3.1 of {{RFC7800}} in which different authenticity confirmation methods can be included.
```
make it more general

Reply:
I formulated this for JWT specifically as a starting point. This is similar to the JWT cnf registry. My current understanding is that we need to define 2 registries for "status" - one for JWT and one for CWT.
I guess I should adjust the title to match it - let's discuss in the editors call about this issue imho.